src/shared/capture/capturefromfile.cpp

```diff
@@ -119,18 +126,40 @@ bool CaptureFromFile::startCapture()
   
     // Read images to buffer in memory:
     imgs_to_load.sort();
-    imgs_it = imgs_to_load.begin();
-    std::list<std::string>::iterator currentImage = imgs_it;
-    while(currentImage != imgs_to_load.end())
-    {
-      int width(-1);
-      int height(-1);
-      rgba* rgba_img = ImageIO::readRGBA(width, height, currentImage->c_str());
-      fprintf (stderr, "Loaded %s \n", currentImage->c_str());
-      images.push_back(rgba_img);
-      heights.push_back(height);
-      widths.push_back(width);
-      ++currentImage;
+    for (auto &currentImage : imgs_to_load) {
```

Comment:
This can be a `const auto&` for specificity, and to help catch accidental modification bugs in the future.